randovania/games/dread/pickup_database/pickup-database.json

```diff
@@ -352,7 +353,8 @@
             "broad_category": "aeion",
             "model_name": "powerup_sonar",
             "offworld_models": {
-                "am2r": "sItemPulseRadarDread"
+                "am2r": "sItemPulseRadarDread",
+                "planets_zebeth": "spr_ITEM_Sensor_Visor"
```

Comment:
I would remove this and thus make it default to the generic offworld model for now.